src/Discovery/DiscoverySummary.tsx

```diff
@@ -21,7 +21,16 @@ const renderAggregation = (aggregation: AggregationConfig, studies: any[] | null
   switch (type) {
   case 'sum': {
     // parse any string representation of an integer
-    fields = fields.map((item) => (typeof item === 'string' ? parseInt(item, 10) || 0 : item));
+    // if it's not a number, return 0 so as not to break the sum
```

Comment:
Nitpick: instead of using a comment here, the code could be wrapped in a function with an explanatory name. Reduces likelihood that code changes and the comment does not.  

Examples: 
```
const checkForNonNumericInput = (input) => {
  ....
}
```
or
```
const handleNonNumericInput = () => {
   ...
}
```

Reply:
used your first suggestion